pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -753,8 +753,8 @@ let
         mktplcRef = {
           name = "vscode-eslint";
           publisher = "dbaeumer";
-          version = "2.2.6";
-          sha256 = "sha256-1yZeyLrXuubhKzobWcd00F/CdU824uJDTkB6qlHkJlQ=";
+          version = "2.4.0";
+          sha256 = "02q5g5p2vy7yxm2l89zvlkfx4rkdyzrrjaa97gl5sf6g88k11igc";
```

Comment:
How did you generate the hash? they should start by "sha256-" now.

Reply:
I used `./path-to-nixpkgs/pkgs/applications/editors/vscode/extensions/update_installed_exts.sh ` This gave me the sha i used.

I can build the package both with that output and the sha you provided

Reply:
Yes, I think "SRI hashes" is preferred according to https://nixos.wiki/wiki/Nix_Hash#Usage

Reply:
Updated to use the SRI hash